processor/src/main/java/org/bindgen/processor/util/ConfUtil.java

```diff
@@ -58,35 +56,58 @@
 	 * So we start there and walk up parent directories looking for
 	 * {@code bindgen.properties} files.
 	 */
-	private static File resolveBindgenPropertiesIfExists(Location location, ProcessingEnvironment env, String fileName) {
-		// Find a dummy /bin/apt/dummy.txt path to start at
-		final String dummyPath;
-		try {
-			// We don't actually create this, we just want its URI
-			FileObject dummyFileObject = env.getFiler().getResource(location, "", "dummy.txt");
-			dummyPath = dummyFileObject.toUri().toString().replaceAll("file:", "");
-		} catch (IOException e1) {
-			return null;
-		}
+	private static InputStream resolveBindgenPropertiesIfExists(Location location, ProcessingEnvironment env, String fileName) {
+	    final String propertiesFilename = "bindgen.properties";
```

Comment:
Why did you add and use `propertiesFilename` (constant)? Why not use parameter `fileName` as previously?

https://github.com/nfekete/bindgen/blob/07327351e85e6f064608cd7928cb4f17a0ebe7af/processor/src/main/java/org/bindgen/processor/util/ConfUtil.java#L75-L78

Reply:
Good catch, thanks!